content/updates/_index.md

```diff
@@ -14,6 +14,8 @@ The update files are the easiest way to apply an update and are the recommended
 approach. The full images are provided in case an update goes wrong or you need
 to completely wipe your Pi for some reason.
 
+If no update files are provided, you should update using the [full image](/kit/pi/sd-card) download.
```

Comment:
This should be "update file" as you are only talking about a singular file in this context.

Reply:
fixed in 33ec8a71acc0d4b1041b49e1612feb5353545513